CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.1.11
```

Comment:
Unfortunately I can't release this yet. I have unstable features on master that are in development. The next release will be a breaking change and will bump to 0.2 but leave this out for now as I will update the change log when the next release is ready to roll out.

Reply:
@ryanheise ok no problem. Would you like me to remove the change log entry & version bump from this PR so that it's still mergeable?

Reply:
Yes please leave this out as I'd rather manage the change log when I do the next release. Thanks.